Remotion/Data/DomainObjects.UnitTests/Factories/BaseConfiguration.cs

```diff
@@ -118,5 +124,29 @@ public FakeDomainObjectsConfiguration GetDomainObjectsConfiguration ()
     {
       return new FakeDomainObjectsConfiguration(_mappingLoaderConfiguration, _storageConfiguration, _queryConfiguration);
     }
+
+    public void DisableConnections ()
+    {
+      foreach (var rdbmsProviderDefinition in _storageConfiguration.StorageProviderDefinitions.OfType<RdbmsProviderDefinition>())
+      {
+        var connectionString = rdbmsProviderDefinition.ConnectionString;
+        if (string.IsNullOrEmpty(connectionString))
```

Comment:
You're using this to skip updating the connection strings. That's not really obvious from reading the code. It would be better to introduce a named flag for this condition. Also, continue vs statement-block: when we have  a nicely-named flag, we don't need `continue`.

About `continue`:
Use `break` when it's not possible to return. Use `continue` if the alternative is really ugly. It's derived from the intention of not using goto-label.